Cargo.toml

```diff
@@ -57,6 +57,9 @@ pattern = []
 # Enable to use simd acceleration.
 # Note that this is deprecated and is a no-op.
 simd-accel = []
+# When testing, run the expensive randomized testing suites in addition
+# to the fast unit tests.
+random-test = []
```

Comment:
I think we probably don't want a feature for this. Features are part of a crate's public API, and this is really an internal implementation detail of a specific kind of test.

We can instead use normal cfg's, which might be set by env vars. We already do this for enabling simd vector optimizations in the `build.rs`.